addons/mail/static/src/discuss/call/common/call_participant_video.js

```diff
@@ -5,7 +5,7 @@ import { useRtc } from "@mail/discuss/call/common/rtc_hook";
 import { Component, onMounted, onPatched, useExternalListener, useRef } from "@odoo/owl";
 
 export class CallParticipantVideo extends Component {
-    static props = ["session"];
+    static props = ["session", "videoStream?"];
```

Comment:
Since your are adding a props, I would take the opportunity to type de component and it's props.